### ISSUE TITLE
Query - Is this extension using the entire whatsapp-web.js code

### ISSUE THREAD

**Issue by KoolPal** [naranarethiya/whatsapp-web-interaction-suite#5]
Hi,

Is your extension using the entire whatsapp-web.js code?

I am asking as there are some issues we observed which seems to have been fixed at whatsapp-web.js but continues to persist here

I was looking for a Fix for url preview in message - which was not working in our WhatsApp web even when we are just replying with a link to some message manually.
This seems to have been fixed here
https://github.com/pedroslopez/whatsapp-web.js/pull/3210/commits/8fa487cd27c07e3f736d635deebc8a5db17874bb

Additionally we noticed that with your extension active, weekdays in chat history appear in Portuguese- like 
Segunda-feira: Monday
Terça-feira: Tuesday
Quarta-feira: Wednesday
Quinta-feira: Thursday
Sexta-feira: Friday
Sábado: Saturday
Domingo: Sunday

This gets fixed once we disable your extension and refresh WA Web.

Can you please check and guide?

Thanks

**Comment by narethiya-godaddy** [comment]
@KoolPal

> Is your extension using the entire whatsapp-web.js code?

No, the extension uses specific parts of the whatsapp-web.js library. However, since the patch they implemented affects the code utilized by this extension, it’s necessary to address and fix that part accordingly.

> Additionally we noticed that with your extension active, weekdays in chat history appear in Portuguese
The extension does not modify or interfere with the WhatsApp Web UI. Could you please share a screenshot of the issue so we can investigate further?

**Comment by naranarethiya** [comment]
@KoolPal

> Is your extension using the entire whatsapp-web.js code?

No, the extension uses specific parts of the whatsapp-web.js library. However, since the patch they implemented affects the code utilized by this extension, it’s necessary to address and fix that part accordingly.

> Additionally we noticed that with your extension active, weekdays in chat history appear in Portuguese

The extension does not modify or interfere with the WhatsApp Web UI. Could you please share a screenshot of the issue so we can investigate further?

**Comment by KoolPal** [comment]
Hi,

I have a bad cold and I just realized I mentioned too many issues in a single thread. Let me break it up so that it is easier to converse.

Thank you very very much for sparing your time for this awesome extension. Really appreciate it.

**Comment by KoolPal** [comment]
I am not sure if you can help about the link preview issue as I dug deeper and found that it is impacting ONLY 1 WA Business number which is on an iphone. Everything works on other WA Business number on Android.

**Comment by KoolPal** [comment]
Let's use thread to discuss what features from the below are available in your extension. Basis your reply, I will see how to best leverage your extension.

https://github.com/pedroslopez/whatsapp-web.js#supported-features

Thanks
